### PR TITLE
(BIDS-2531) add filter for online/offline, fix all count

### PR DESCRIPTION
**Pull request by gocoffeecup**
### <samp>🤖 Generated by Copilot at ef36a11</samp>

This pull request adds the ability to filter and view validators by their online or offline status on the web interface and the API. It modifies the `templates/validators.html` file and the `parseValidatorsDataQueryParams` function in the `handlers/validators.go` file.

### CODE REVIEW

File: templates/validators.html [hunk 1]
```diff
@@ -15,17 +15,22 @@
...
-            <a class="dropdown-item" href="#deposited" data-filter-validators="deposited"><span>Deposited</span> <span>${depositedCount}</span></a>
-            <a class="dropdown-item" href="#pending" data-filter-validators="pending"><span>Pending</span> <span>${pendingCount}</span></a>
+            <a class="dropdown-item" href="#online" data-filter-validators="online"><span>Online</span> <span>${onlineCount}</span></a>
+            <a class="dropdown-item" href="#offline" data-filter-validators="offline"><span>Offline</span> <span>${offlineCount}</span></a>
             <a class="dropdown-item" href="#active" data-filter-validators="active"><span>Active</span><span> <span class="text-success">${activeOnlineCount}</span> | <span class="text-danger">${activeOfflineCount}</span></span></a>
             <a class="dropdown-item" href="#slashing" data-filter-validators="slashing"><span>Slashing</span><span> <span class="text-success">${slashingOnlineCount}</span> | <span class="text-danger">${slashingOfflineCount}</span></span></a>
+            <a class="dropdown-item" href="#deposited" data-filter-validators="deposited"><span>Deposited</span> <span>${despositedCount}</span></a>
```
Comment (Eisei24) [inline templates/validators.html:32]: **issue:** `de**s**positedCount` typo here that causes an empty datatable.

**Review by Eisei24 — CHANGES_REQUESTED** [review]
The typo needs to be fixed otherwise looking good.

**Review by Eisei24 — APPROVED** [review]
lgtm

**Comment by D13ce** [comment]
Please ignore the "Revert" commit :frowning:

**Pull request merged**